robocorp-code/src/robocorp_code/inspector/java/robocorp_java/_locators.py

```diff
@@ -0,0 +1,73 @@
+from typing import List, Optional, Sequence, Union
+
+from JABWrapper.context_tree import (  # type: ignore
+    ContextNode,
+    SearchElement,
+)
+
+IntegerLocatorTypes = [
+    "x",
+    "y",
+    "width",
+    "height",
+    "row",
+    "col",
+    "indexInParent",
+    "childrenCount",
+    "ancestry",
+]
+
+
+def parse_locator(locator: str, strict_default=False):
```

Comment:
Maybe should use the same parse code which is used by robocorp-windows? (future work probably...)

Reply:
That's part of the wrapper - it's vendor code. 
Most probably future work as if the parsing changes the validation isn't the same as the library.